ansible/group_vars/edusharing.yml

```diff
@@ -177,6 +178,21 @@ alfresco_ccmodel_add_custom_properties:
 # 
 edu_custom_licenses:
 
+
+# define config for classifications keywords
+# they are also some default configurations in /roles/edu-sharing/default/main/yml
+# activate the classification-keyword (true/false)
+activate_classification_keywords: false
+# activate the classification-keyword update (true/false)
+activate_classification_keywords_update: false
+# must be public folder: example: /oer/classification-keywords
+classification_keywords_dir: 
+# define the classification-keyword file name example: classification_keywords.xml
+classification_keywords_file: 
```

Comment:
Is this likely to be changed by the user? If not, remove to avoid unnecessarily complex configuration.